Library/ViewModels/ProjectActivityUpdateCellViewModel.swift

```diff
@@ -88,16 +88,16 @@ ProjectActivityUpdateCellViewModelInputs, ProjectActivityUpdateCellViewModelOutp
   public var outputs: ProjectActivityUpdateCellViewModelOutputs { return self }
 }
 
-private func updateNumber(activity activity: Activity) -> String {
+private func updateNumber(activity: Activity) -> String {
   guard let update = activity.update else { return "" }
   return Strings.dashboard_activity_update_number_posted_time_count_days_ago(
     space: "\u{00a0}",
-    update_number: Format.wholeNumber(update.sequence ?? 0),
+    update_number: Format.wholeNumber(update.sequence),
```

Comment:
Does this format as expected if the sequence number is nil? Not sure if there's a test for that...